src/renderer/index.js

```diff
@@ -61,10 +63,17 @@ ipc.on('will-quit', () => {
   store.dispatch(setUIState('isExiting', true));
 });
 
+// show message, when locale changed
+ipc.on('locale-changed', (e, payload) => {
+  alert(payload);
```

Comment:
Can you use the `showDialog` function in `renderer/system/dialog.js`?

Reply:
Also, why does the user need to relaunch the program? wont react-intl change the texts in place?

Reply:
Jep

Reply:
Because I can't find a solution to update the whole electron menu.

Reply:
@ph1p We do that when a new archive is added. For example I call it everytime the store gets updated: `store.subscribe(debounce(() => setupMenu(store), 500));`. So this should also update the language already if everything's setup correctly

Reply:
Ok cool. I'll try it :)

Reply:
Yeah, It works!